explainer.md

```diff
@@ -667,18 +693,31 @@ An example of how this could look:
 ![](images/image19.png)
 
 <!-- TOC --><a name="complexity"></a>
-### Complexity 
-Most of the implementation complexity of the PEPC lies in the annoyance reduction mechanisms. As previously mentioned, the main security surface is the "Confirmation UI" which is straightforward to implement. 
 
-This proposal describes a deliberately conservative set of annoyance reduction mechanisms with the aim to discovering in [developer trials](#developer-trials) which restrictions will be infeasible for users, developers or implementors. 
+### Complexity
+
+Most of the implementation complexity of the PEPC lies in the annoyance
+reduction mechanisms. As previously mentioned, the main security surface is the
+"Confirmation UI" which is straightforward to implement.
+
+This proposal describes a deliberately conservative set of annoyance reduction
+mechanisms with the aim to discovering in [developer trials](#developer-trials)
+which restrictions will be infeasible for users, developers or implementors.
 
 <!-- TOC --><a name="implementor-portability-internationalization-upkeep"></a>
+
 ### Implementor portability, internationalization & upkeep
-Most browsers already have user recognizable iconography for common permissions such Camera/Microphone or Location and the PEPC can share strings used in the existing permission journey. 
 
-Developers will need to handle the `onvalidationstatuschange` event, which helps to future proof websites against unexpected changes in browser implementor validation criteria. 
+Most browsers already have user recognizable iconography for common permissions
+such Camera/Microphone or Location and the PEPC can share strings used in the
+existing permission journey.
+
+Developers will need to handle the `onvalidationstatuschange` event, which helps
```

Comment:
Covers both "onvalidationstatuschange" and `.addEventListener("validationstatuschange"...)`:

```suggestion
Developers will need to handle the `"validationstatuschange"` event, which helps
```